wsmatrix@martin.zurowietz.de/stylesheet.css

```diff
@@ -0,0 +1,21 @@
+.workspace-switcher {
```

Comment:
I'd rather leave the styling to the themes. The workspace switcher without thumbnails should look exactly like the default switcher, only with a 2D layout. The active workspace should be highlighted just the same, too, but without the up/down arrows (if there is an appropriate class for that).

Reply:
there is no label style for the workspace name, do you think it's better to hard code it or leave it in the stylesheet? or you know some good alternative style to that?

Reply:
IMO the color should be left to the theme as well. As font size I'd use the default system font size but I couldn't find how to apply that. So `10em` should be ok for now. The font rendering doesn't look very good for some reason, though.

![Screenshot from 2019-04-29 20-09-23](https://user-images.githubusercontent.com/2457311/56917111-f21da080-6aba-11e9-88b9-9b554ce3d79e.png)

Reply:
I merged the new master and took care of the other items but I let this one open for discussion.

Reply:
great job!!
I made a new commit to remove the stylesheet and hardcode font-size to 10em
this by default will take the color of the theme, I tried many themes and they are all working fine.

the reason behind bad rendering is probably the scaled widget
it takes the width and height of `workarea`, maybe the width and height should be calculated using `workarea` and `number of grid rows`, and `number of grid columns`
I will take a look at it